packages/bitcore-node/src/config.ts

```diff
@@ -62,7 +62,7 @@ const Config = function(): ConfigType {
     dbPass: process.env.DB_PASS || '',
     numWorkers: cpus().length,
     chains: {},
-    modules: ['./bitcoin', './bitcoin-cash', './ethereum'],
```

Comment:
Revert this line and set it in your bitcore.config.json

Reply:
reverted